test/coverlet.collector.tests/CoverletSettingsParserTests.cs

```diff
@@ -72,6 +72,36 @@ public void ParseShouldCorrectlyParseConfigurationElement()
             Assert.True(coverletSettings.SingleHit);
         }
 
+        [Fact]
+        public void ParseShouldCorrectlyParseMultipleFormats()
```

Comment:
add also test wiht strange cases like
[InlineData(" , json")]
[InlineData(" , json, ")]
[InlineData(" , json, , cobertura ")]
preserving spaces and check that result are ok

Reply:
Done